packages/contracts/contracts/optimistic-ethereum/iOVM/bridge/messaging/iOVM_L1CrossDomainMessenger.sol

```diff
@@ -45,20 +45,4 @@ interface iOVM_L1CrossDomainMessenger is iAbs_BaseCrossDomainMessenger {
         uint256 _messageNonce,
         L2MessageInclusionProof memory _proof
     ) external;
-
-    /**
-     * Replays a cross domain message to the target messenger.
-     * @param _target Target contract address.
-     * @param _sender Original sender address.
-     * @param _message Message to send to the target.
-     * @param _messageNonce Nonce for the provided message.
-     * @param _gasLimit Gas limit for the provided message.
-     */
-    function replayMessage(
```

Comment:
I think we should discuss `replayMessage` in a separate issue and perhaps remove it in a different PR. cc @karlfloersch but I think we may actually need `replayMessage` for our upgrade mechanism.